sdk/docs/core/README.md

```diff
@@ -106,10 +106,11 @@ Log classifications allow your application to select which specific log messages
       printf("%.*s\n", az_span_size(message), az_span_ptr(message));
    }
 
+   az_log_classification const log_classifications[] = { AZ_LOG_HTTP_REQUEST, AZ_LOG_HTTP_RESPONSE, AZ_LOG_END_OF_LIST };
```

Comment:
```suggestion
   static az_log_classification const log_classifications[] = { AZ_LOG_HTTP_REQUEST, AZ_LOG_HTTP_RESPONSE, AZ_LOG_END_OF_LIST };
```

Reply:
Thank you Ahson, good catch, I had this change, but didn't save the changes to file before git commit. I am also adding static for the log callback function.